<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Finestra</class>
 <widget class="QMainWindow" name="Finestra">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Windows 10 tools</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icon.ico</normaloff>../icon.ico</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="Spazio">
   <property name="maximumSize">
    <size>
     <width>640</width>
     <height>480</height>
    </size>
   </property>
   <widget class="QLabel" name="info">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>441</width>
      <height>301</height>
     </rect>
    </property>
    <property name="text">
     <string>Info</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>360</y>
      <width>561</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="toolWindowsUpdate">
         <property name="text">
          <string>Tools Windows Update</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="antivirusBnt">
         <property name="text">
          <string>Antivirus</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="cmdCommands">
         <property name="text">
          <string>CMD commands</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="misc">
         <property name="text">
          <string>Misc</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
